helpers/all.js

```diff
@@ -9,19 +9,13 @@ const _ = require('lodash');
  * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
  */
 const factory = () => {
-    return function() {
-        var args = [], opts, result;
+    return function(...args) {
 
-        // Translate arguments to array safely
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
-        }
-
-        // Take the last argument (content) out of testing array
-        opts = args.pop();
+        // Take the last arg out of args array
+        const opts = args.pop();
```

Comment:
why we need to add const?

Reply:
Just did because my other PR looks like that.

Reply:
@bc-williamkwon Could you clarify why we need const here at all?

Reply:
This is basically the same file as last.  Const is preferrable to use in es6 and above.  One wouldn't be seeing some parts of the changes in this file if the other pr were to be merged.